learn/security/tenant_tokens.md

```diff
@@ -117,7 +117,7 @@ The token payload contains most of the relevant token data. It must be an object
 ```json
 {
   "exp": 1646756934,
-  "apiKeyPrefix": "12345678",
+  "apiKeyUid": "12345678",
```

Comment:
The value for the example should display a real uuid v4 like `74c9c733-3368-4738-bbe5-1d18a5fecb37` (the value is taken from a previous API key displayed in [master_api_keys.md](https://github.com/meilisearch/documentation/pull/1738/files#diff-d40e36ee90d9727237e72205b40af7ea904d5d0ce0b277774e7c5ebe8501bdf3))